### PR TITLE
Added script to trigger aggregation in a Data pipline using Delta lak…

### PR DESCRIPTION
**Pull request by shivahari**
Added a script to:
- Fetch the refined `Trello` card data from the `Silver` table
- Filter the data based on a `member` contributed to a `Trello` card
- Build a `Gold` table with the filtered data

### CODE REVIEW

**Comment by sravantit25** [comment]
> Added a script to:
> 
> * Fetch the refined `Trello` card data from the `Silver` table
> * Filter the data based on a `member` contributed to a `Trello` card
> * Build a `Gold` table with the filtered data

@shivahari Impressive work, Shiva! Thanks to your work on this task, I got to know good standards of writing Python code, the way you initialized methods, property and the whole structure as such. Good learning for me from you as always! 

I am suggesting two changes: 
1. rename your script to shiva_cards_gold.py (since everyone else will also have their own scripts, need to be distinguishable)
2. Can you refine/filter the data to fetch only few columns which would be sufficient for gold table -
"id", "name","LastUpdated", "board_name", "card_members"

File: jobs/build_gold_table.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+            :self.member: Member contributed to the Trello card
+            :self.start_date: Date from which the member started contributing
+            :self.edn_date: Date until which the member contributed
+        return:
+            :cards: Trello cards Data Frame
+        """
+        try:
+            self.logger.info(f"Gettings cards for member {self.member}")
+            cards = self.spark.read.format("delta").load(self.silver_table)
+            cards = cards.where(col("card_members").contains(self.member))
```
Comment (sravantit25) [inline jobs/build_gold_table.py]: Can you refine/filter the data to fetch only few columns which would be sufficient for gold table - 
"id", "name","LastUpdated", "board_name", "card_members"

File: jobs/build_gold_table.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
+"""
```
Comment (sravantit25) [inline jobs/build_gold_table.py]: Can you rename the script to shiva_cards_gold.py (or something similar).

**Review by sravantit25 — CHANGES_REQUESTED** [review]

File: jobs/build_gold_table.py (hunk 1 above)
Comment (shivahari) [reply jobs/build_gold_table.py]: I have added a step to filter the data based on a `TrelloCardProperties` Enum

File: jobs/build_gold_table.py (hunk 2 above)
Comment (shivahari) [reply jobs/build_gold_table.py]: I have renamed the file as `shiva_cards_gold.py`

File: jobs/build_gold_table.py (hunk 1 above)
Comment (sravantit25) [reply jobs/build_gold_table.py]: Thanks for making the changes Shiva. Looks good. Merging the PR.

**Pull request merged**